src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -476,50 +507,133 @@ public void testUpdateDistinctAttributeSettings() throws Exception {
+        Index index = createIndex("testResetDistinctAttributeSettings");
         String initialDistinctAttribute = index.getDistinctAttributeSettings();
         String newDistinctAttribute = "title";
 
         index.waitForTask(index.updateDistinctAttributeSettings(newDistinctAttribute).getUid());
         String updatedDistinctAttribute = index.getDistinctAttributeSettings();
 
-        assertEquals(newDistinctAttribute, updatedDistinctAttribute);
-        assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
-
         index.waitForTask(index.resetDistinctAttributeSettings().getUid());
         String distinctAttributeAfterReset = index.getDistinctAttributeSettings();
 
+        assertEquals(newDistinctAttribute, updatedDistinctAttribute);
+        assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
+
         assertNotEquals(updatedDistinctAttribute, distinctAttributeAfterReset);
         assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
     }
 
+    /** Tests of the typo tolerance setting methods */
     @Test
-    @DisplayName("Test reset ranking rules when null value is passed")
-    public void testUpdateRankingRuleSettingsUsingNull() throws Exception {
-        Index index = createIndex("testUpdateRankingRuleSettingsUsingNull");
-        String[] initialRankingRule = index.getRankingRuleSettings();
-        String[] newRankingRules = {
-            "typo",
-            "words",
-            "sort",
-            "proximity",
-            "attribute",
-            "exactness",
-            "release_date:desc",
-            "rank:desc"
-        };
+    @DisplayName("Test get typo tolerance settings by uid")
+    public void testGetTypoTolerance() throws Exception {
+        Index index = createIndex("testGetTypoTolerance");
+        Settings initialSettings = index.getSettings();
+        TypoTolerance initialTypoTolerance = index.getTypoToleranceSettings();
 
-        index.waitForTask(index.updateRankingRuleSettings(newRankingRules).getUid());
-        String[] newRankingRule = index.getRankingRuleSettings();
+        assertEquals(
+                initialSettings.getTypoTolerance().getDisableOnWords().length,
+                initialTypoTolerance.getDisableOnWords().length);
+        assertEquals(
+                initialSettings.getTypoTolerance().getDisableOnAttributes().length,
+                initialTypoTolerance.getDisableOnAttributes().length);
+        assertEquals(
+                initialSettings.getTypoTolerance().isEnabled(), initialTypoTolerance.isEnabled());
+        assertTrue(
+                initialTypoTolerance.getMinWordSizeForTypos().containsKey("oneTypo")
+                        && initialTypoTolerance.getMinWordSizeForTypos().get("oneTypo") != null);
+        assertTrue(
+                initialTypoTolerance.getMinWordSizeForTypos().containsKey("twoTypos")
+                        && initialTypoTolerance.getMinWordSizeForTypos().get("twoTypos") != null);
```

Comment:
You can use https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertNotNull-java.lang.Object- for this case :)